superset/templates/superset/basic.html

```diff
@@ -107,5 +108,8 @@ <h4 class="modal-title"></h4>
         {% endwith %}
       {% endif %}
     {% endblock %}
+    {% if walkMeEnabled %}
+    <script type="text/javascript">(function() {var walkme = document.createElement('script'); walkme.type = 'text/javascript'; walkme.async = true; walkme.src = 'https://cdn.walkme.com/users/fe8e118680a14f439c78916022601810/test/walkme_fe8e118680a14f439c78916022601810_https.js'; var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(walkme, s); window._walkmeConfig = {smartLoad:true}; })();</script>
+    {% endif %}
```

Comment:
js link comment same as before

Reply:
No dependency is available on npm for `walkme`